src/medicalpage/SingleMedication.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { EditStatusIcon } from './EditStatus';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
+  faTrash,
+  faEdit,
+  faCheckSquare
+} from '@fortawesome/free-solid-svg-icons';
+import Input from '../components/Input';
+
+library.add(faTrash, faEdit, faCheckSquare);
+
+const StyledInput = styled.input`
+  width: 100%;
+`;
+
+const SingleMedicationContainer = styled.li`
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 1fr 3fr 2fr 1fr 1fr;
+  list-style: none;
+`;
+
+const TimeContainer = styled.div`
+  width: 100%;
+  grid-column: 1;
+`;
+
+const MedicineContainer = styled.div`
+  width: 100%;
+  grid-column: 2;
+`;
+
+const DosageContainer = styled.div`
+  width: 100%;
+  grid-column: 3;
+`;
+
+const DeleteIcon = styled(FontAwesomeIcon)`
+  grid-column: 4;
+`;
+
+const DeleteButton = styled.button`
+  grid-column: 4;
+`;
+const StyledEditStatusIcon = styled(EditStatusIcon)`
```

Comment:
These are a lot of components. It could help visibility to extract these to a `SingleMedicationComponents.js`and import them like `import {TimeContainer, DeleteIcon,...} from './SingleMedicationComponents'`

Reply:
Thats a great idea. Thanks!